### PR TITLE
Kv doc update

### LINKED ISSUE #5898: Update KeyVault Admin Root README
Update the root `README.md` of the Azure SDK for C++ KayVault Admin library to follow the template in [this issue.](https://github.com/Azure/azure-sdk-pr/issues/1745)

### LINKED ISSUE #5899: Update KeyVault Certificates Root README
Update the root `README.md` of the Azure SDK for C++ KeyVault Certificates library to follow the template in [this issue.](https://github.com/Azure/azure-sdk-pr/issues/1745)

### LINKED ISSUE #5900: Update KeyVault Keys Root README
Update the root `README.md` of the Azure SDK for C++ KeyVault Keys library to follow the template in [this issue.](https://github.com/Azure/azure-sdk-pr/issues/1745)

### LINKED ISSUE #5901: Update KeyVault Secrets Root README
Update the root `README.md` of the Azure SDK for C++ KeyVault Secrets library to follow the template in [this issue.](https://github.com/Azure/azure-sdk-pr/issues/1745)

### PR DESCRIPTION
**Pull request by gearama**
closes #5898 
closes #5899 
closes #5900
closes #5901 
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#pull-requests).

- [ ] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [ ] Doxygen docs
- [ ] Unit tests
- [ ] No unwanted commits/changes
- [ ] Descriptive title/description
  - [ ] PR is single purpose
  - [ ] Related issue listed
- [ ] Comments in source
- [ ] No typos
- [ ] Update changelog
- [ ] Not work-in-progress
- [ ] External references or docs updated
- [ ] Self review of PR done
- [ ] Any breaking changes?

### CODE REVIEW

**Comment by ronniegeraghty** [comment]
Before the first Azure CLI command (line 24) add the `az login` command just as a note that it's needed and that the AzureDefaultCrednetial will use it.

File: sdk/keyvault/azure-security-keyvault-administration/README.md [hunk 1]
```diff
@@ -92,8 +118,10 @@ We guarantee that all client instance methods are thread-safe and independent of
 ## Examples 
 
 For detailed samples please review the code provided.
```
[the remark this answers is not in the document]
Comment (gearama) [reply sdk/keyvault/azure-security-keyvault-administration/README.md:122]: we use them all over the place it gives an idea that here be samples

File: sdk/keyvault/azure-security-keyvault-certificates/README.md [hunk 2]
```diff
@@ -4,52 +4,75 @@ Azure Key Vault is a cloud service that provides secure storage and automated ma
...
+To install the Azure \<Service-Name> package via vcpkg:
+To add the Azure \<Service-Name> package to your vcpkg enter the following command (We'll also add the Azure Identity library for authentication):
+
+```batch
+vcpkg add port azure-security-keyvault-certificates-cpp azure-identity-cpp
+```
+
+Then, add the following in your CMake file:
+
+```CMake
```
Comment (ronniegeraghty) [inline sdk/keyvault/azure-security-keyvault-certificates/README.md:40]:
```suggestion
```CMake
find_package(azure-identity-cpp CONFIG REQUIRED)
```

File: sdk/keyvault/azure-security-keyvault-administration/README.md [hunk 3]
```diff
@@ -78,6 +73,39 @@ az keyvault role assignment create --hsm-name <your-managed-hsm-name> --role "Ma
 
 Please read [best practices][best_practices] for properly securing your managed HSM.
 
+### Install the package
+
+he easiest way to acquire the C++ SDK is leveraging the vcpkg package manager and CMake. See the corresponding [Azure SDK for C++ readme section][azsdk_vcpkg_install]. We'll use vcpkg in manifest mode. To start a vcpkg project in manifest mode use the following command at the root of your project:
```
Comment (ronniegeraghty) [inline sdk/keyvault/azure-security-keyvault-administration/README.md:78]:
```suggestion
The easiest way to acquire the C++ SDK is leveraging the vcpkg package manager and CMake. See the corresponding [Azure SDK for C++ readme section][azsdk_vcpkg_install]. We'll use vcpkg in manifest mode. To start a vcpkg project in manifest mode use the following command at the root of your project: 
```

File: sdk/keyvault/azure-security-keyvault-keys/README.md [hunk 4]
```diff
@@ -9,11 +9,53 @@ you to safeguard cryptographic keys for your cloud applications using FIPS 140-2
...
+To install the azure-security-keyvault-keys-cpp package via vcpkg:
+To add the azure-security-keyvault-keys-cpp package to your vcpkg enter the following command (We'll also add the Azure Identity library for authentication):
+
+```batch
+vcpkg add port azure-security-keyvault-keys-cpp azure-identity-cpp
+```
+
+Then, add the following in your CMake file:
+
+```CMake
```
Comment (ronniegeraghty) [inline sdk/keyvault/azure-security-keyvault-keys/README.md:43]:
```suggestion
```CMake
find_package(azure-identity-cpp CONFIG REQUIRED)
```

**Review by ronniegeraghty — CHANGES_REQUESTED** [review]

**Review by ronniegeraghty — APPROVED** [review]

**Pull request merged**